src/LoginForm/LoginForm.css

```diff
@@ -1,15 +1,128 @@
+.main-content {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 100vh;
+  background-image: url("../assets/bluciferHERO.jpg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  overflow: hidden;
+}
+
+.title {
+  -webkit-text-stroke-width: 2px;
+  -webkit-text-stroke-color: #FB4F14;
+  margin: 2% 0 0 0;
+  font-weight: 900;
+  font-size: 2.5em;
+  color: #ffffff;
+  text-shadow: 0 0 3px #000;
+}
+
+.text-center {
+	color:#fff;	
+  font-size: 1.5em;
+  margin: 0 0 20% 0;
+  display: flex;
+  justify-content: center;
+}
+
 .login-form {
   display: flex;
   flex-direction: column;
+  background-color: rgba(0, 0, 0, 0.75);
+	border-radius: 3px;
+  padding: 5% 6%;
+  margin: 2% 0 0 0;
+  box-shadow: 0 0 3px #000;
+}
+
+.input-container {
+	position: relative;
+  margin-bottom: 25px;
+  font-weight: bolder;
+}
+
+.input-container label{
+	position: absolute;
+	top: 0px;
+	left: 0px;
+	font-size: 16px;
+	color:#fff;	
+	transition: all 0.5s ease-in-out;
+}
+
+.input-container input{ 
+  border: 0;
+  border-bottom: 1px solid #555;  
+  background: transparent;
+  width: 100%;
+  padding: 6% 0 0.5% 0;
+  font-size: 20px;
+  color:#fff;
+}
+
+.input-container input:focus { 
+  border: none;	
+  outline: none;
+  border-bottom: 1px solid #FB4F14;	
+}
+
+.input-container input:focus ~ label,
+.input-container input:valid ~ label{
+  margin: -2% 0 0 0;
+	font-size:12px;
+}
+
+.purpose-buttons {
+  margin: -10% 0 0 0;
+  display: flex;
+  justify-content: space-around;
+}
+
+.purpose-btn {
+	color:#fff;
```

Comment:
The tabbing in this file is suspicious

Reply:
I think it might have to do with a setting between 2 space tabbing and 4 space tabbing?